perllib/Open311/GetServiceRequestUpdates.pm

```diff
@@ -110,12 +110,13 @@ sub update_comments {
 
             if ( !$c->first ) {
                 my $state = $open311->map_state( $request->{status} );
+                my $external_status_code = $request->{external_status_code};
                 my $comment = $self->schema->resultset('Comment')->new(
                     {
                         problem => $p,
                         user => $self->system_user,
                         external_id => $request->{update_id},
-                        text => $self->comment_text_for_request($request, $p, $state),
+                        text => $self->comment_text_for_request($request, $p, $state, $external_status_code),
```

Comment:
This is added here, but the fn isn't updated until a different commit (so I guess this line change should wait until then, or you might be squashing them together in which case doesn't matter!).

Reply:
Oops, yes, that should be in the later commit.